Maestro.json

```
@@ -267,6 +273,312 @@
         }
       ]
     },
+    {
+      "Identifier": "YMML",
+      "FeederFixes": ["ARBEY", "BOYSE", "BOYSE/V", "BOOIN", "BOOIN/V", "RAZZI/AV", "RAZZI/M", "PORTS", "ALAXO", "ALAXO/V", "ALAXO/P"],
```

The feeder fixes with `/V` and `/M` etc. won't work. These need to be the **exact** name of the feeder fix.
```suggestion
      "FeederFixes": ["ARBEY", "BOYSE", "BOOIN", "RAZZI", "PORTS", "ALAXO"],
```

---

Am stupid, missed that.